src/stdune/csexp.ml

```diff
@@ -74,3 +74,5 @@ let to_string sexp =
   let buf = buffer () in
   to_buffer sexp ~buf;
   Buffer.contents buf
+
+let of_string string = parse (Stream.of_string string)
```

Comment:
I think this function should fail if there's a non-empty suffix remaining after parsing. As it stands, I think it ignores all input after the first valid csexp.